DataFormats/NanoAOD/src/classes_def.xml

```diff
@@ -9,6 +9,9 @@
     <class name="nanoaod::FlatTable::RowView" transient="true" />
     <class name="edm::Wrapper<nanoaod::FlatTable>" />
 
+    <class name="edm::Wrapper<std::vector<nanoaod::FlatTable>>" />
+    <class name="std::vector<nanoaod::FlatTable>" />
```

Comment:
where is this used?
I thought that a table is already a flat vector supporting many types; why make vectors of it

Reply:
This is a long-running discussion in XPOG. An earlier implementation relied on this, and we plan to partially revert back to it. Doing this is part of the WIP tag, but it should come shortly.

Currently we produce one flat table per weight type, to avoid such modifications, but these causes issues for reading single weights from a very large vector branch (~1000 for PDFs with all kept). We want to instead produce a variable number of tables per type (e.g., 10 different PDF tables). We don't know how many tables there will be until parsing the LHE header or reading the products, so we produce a variable number and then flatten the vector at the NanoAOD EDM --> flat stage. the corresponding code for this is in the NanoAODOutputModule/TableOutputbranches code.